docs/v3.x/installation/docker.md

```diff
@@ -76,22 +75,21 @@ services:
       DATABASE_PORT: 27017
       DATABASE_USERNAME: strapi
       DATABASE_PASSWORD: strapi
-    links:
-      - mongo:mongo
     volumes:
       - ./app:/srv/app
     ports:
       - '1337:1337'
+    depends_on:
+      - mongo
 
   mongo:
     image: mongo
     environment:
+      MONGO_INITDB_DATABASE: strapi
       MONGO_INITDB_ROOT_USERNAME: strapi
       MONGO_INITDB_ROOT_PASSWORD: strapi
     volumes:
-      - ./data/db:/data/db
-    ports:
-      - '27017:27017'
+      - ./db:/data/db
```

Comment:
I would use ./data so it is consistent between every examples :)

Reply:
Sorry for the late reply. Thank you for taking your time to change it. 👍